Dockerfile

```diff
@@ -49,82 +50,82 @@ LABEL org.label-schema.schema-version="1.0" \
       org.label-schema.vendor="inventree" \
       org.label-schema.name="inventree/inventree" \
       org.label-schema.url="https://hub.docker.com/r/inventree/inventree" \
-      org.label-schema.vcs-url=${INVENTREE_GIT_REPO} \
-      org.label-schema.vcs-branch=${INVENTREE_GIT_BRANCH} \
-      org.label-schema.vcs-ref=${INVENTREE_GIT_TAG}
```

Comment:
@SchrodingersGat don't we need this tag?

Reply:
The git commit information is now passed through to the docker image at build time - the "production" image does not contain the `.git` directory any longer

Reply:
The tags are exposed if the image is up - right? A user once told me that he uses the image labels for kubernetes automation.